packages/react/ds/src/alert/alert.tsx

```diff
@@ -0,0 +1,103 @@
+'use client';
+import { useState, type ReactNode } from 'react';
+import { tv, type VariantProps } from 'tailwind-variants';
+import { cn } from '../cn.js';
+import { Icon } from '../icon/icon.js';
+import { IconButton } from '../icon-button/icon-button.js';
+
+const alertVariants = tv({
+  slots: {
+    base: 'gi-alert-base',
+    container: 'gi-alert-container',
+    heading: 'gi-alert-title',
+    dismiss: 'gi-alert-dismiss',
+  },
+  variants: {
+    variant: {
+      info: {
+        base: 'gi-alert-info',
+      },
+      danger: {
+        base: 'gi-alert-danger',
+      },
+      success: {
+        base: 'gi-alert-success',
+      },
+      warning: {
+        base: 'gi-alert-warning',
+      },
+    },
+  },
+  defaultVariants: {
+    variant: 'info',
+  },
+});
+
+type AlertProps = {
+  variant?: VariantProps<typeof alertVariants>['variant'];
+  title: string;
+  children?: ReactNode;
+  className?: string;
```

Comment:
I know that in the past we added `className` but since we decided to remove any possible customizations from the users, we should remove it.

Reply:
maybe keeping it would be helpfull to build the Toast?

Reply:
Just had a conversation with @pregno earlier and the suggestion was that we should remove the className prop across all components so that users will have limited flexibility when it comes to styling as it might affect accessibility standards. The ticket to tackle that is [here](https://dev.azure.com/OGCIO-Digital-Services/Digital%20Services%20Programme/_workitems/edit/23203)